tfhe/src/integer/server_key/radix/scalar_mul.rs

```diff
@@ -255,6 +255,7 @@ impl ServerKey {
         if !self.is_small_scalar_mul_possible(ctxt, scalar) {
             self.full_propagate(ctxt);
         }
+        // assert!(self.is_small_scalar_mul_possible(ctxt, scalar));
```

Comment:
same question everywhere, why is it commented here and in this case even more so given it's not commented below